### DISCUSSION TITLE
Why does semantic-convention `aws-sdk.yml` redefine existing attributes?

### DISCUSSION THREAD

**Discussion by weyert** [open-telemetry/opentelemetry-specification#1607] [Q&A]
I am trying to generate the semantic attributes for the `opentelemetry-js`-project but when I generate the file using the `semconv`-tool the file `aws-sdk.yml` seems to override the description for the `rpc.system` items defined in `rpc.yaml` file.

May I ask why `aws-sdk.yml`-file is redefining these? 

See:
https://github.com/open-telemetry/opentelemetry-specification/blob/412fb5774cefc32ff9b7f47ef80e9e7d245e1e2e/semantic_conventions/trace/instrumentation/aws-sdk.yml#L13

and

See:
https://github.com/open-telemetry/opentelemetry-specification/blob/412fb5774cefc32ff9b7f47ef80e9e7d245e1e2e/semantic_conventions/trace/rpc.yaml#L13

**Comment by anuraaga** [comment]
This was meant to generate the markdown files properly, but seems to have had a negative impact on code generation. I filed https://github.com/open-telemetry/build-tools/issues/41 to fix the code generation.
